addons/l10n_it/models/account_report.py

```diff
@@ -10,6 +10,8 @@ class AccountReportExpression(models.AbstractModel):
     def _get_carryover_target_expression(self, options):
         if self.report_line_id.code == 'VP14b' and fields.Date.from_string(options['date']['date_to']).month == 12:
             # For this line, if we are between two years, we want to carry over to vp9 instead of the line set in the XML file (vp8)
+            if self.env.ref("l10n_it.tax_monthly_report_line_vp14b_carryover") in self.report_line_id.expression_ids:
```

Comment:
`raise_if_not_found=False`

But why not this?
```suggestion
            if self == self.env.ref('l10n_id.tax_monthly_report_line_vp14b', raise_if_not_found=False):
```

Reply:
Indonesia?